metacatalog_api/templates/static/metacatalog.js

```
@@ -0,0 +1,258 @@
+// currently I can't find a better way for this:
+function getEl (selector) {
+    const el = htmx.find(selector);
+    if (!!el) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)